templates/partners/desktop.html

```diff
@@ -7,143 +7,171 @@
+    </div>
+    <div class="col-4 col-medium-2 p-card--strip-thin">
+      <h3 class="p-heading--4">The key to corporate fleet</h3>
+      <p>With 100s of certified models, ranging from low-cost tower PCs to AI workstations</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip has-accent-border--top-right is-deep">
+  <div class="u-fixed-width p-logo-section">
+    <h2 class="p-logo-section__title p-muted-heading u-align-text--center">A selection of our Desktop partners</h2>
+    <ul class="p-logo-section__items">
+      {% for partner in partners %}
+      <li class="p-logo-section__item">
+        <img class="p-logo-section__logo" src="{{ partner.logo }}" alt="{{ partner.slug}}">
+      </li>
+      {% endfor %}
+    </ul>
+    <p class="u-align-text--center">
+      <a href="/partners/find-a-partner?filters=desktop">See all Desktop partners&nbsp;&rsaquo;</a>
+    </p>
+  </div>
+</section>
+
+
+<section class="p-strip has-accent-border--top-left is-deep">
+  <div class="row" style="margin-bottom: 3rem;">
+    <h2>Buy a certified Ubuntu PC</h2>
+    <p>We've been working with partners like Dell, HP and Lenovo for years to ensure that Ubuntu devices reach market as effectively as possible. With the help of these partners, Ubuntu machines are distributed around the world and can be found in both their exclusive and partner stores.</p>
+  </div>
+  <div class="row">
+    <div class="col-4 p-card has-cta">
+      <div class="p-card__content">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/d258d23b-2018-logo-dell.svg",
+          alt="Dell",
+          width="80",
+          height="80",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "u-hide--small u-hide--medium"}
+          ) | safe
+        }}
+        <p>Canonical and Dell work together to deliver Ubuntu on systems from entry-level laptops to high-end developer machines.</p>
+        <p><a href="https://www.delltechnologies.com/en-gb/ai-technologies/index.htm">See the Ubuntu certified AI-ready workstations&nbsp;&rsaquo;</a></p>
+      </div>
+      <a href="http://www.dell.com/linux" class="p-button--positive p-card__cta"><span>Shop for Dell with Ubuntu</span></a>
+    </div>
+    <div class="col-4 p-card has-cta">
+      <div class="p-card__content">
+        {{ image (
+          url="https://assets.ubuntu.com/v1/f627fbfb-hp.svg",
+          alt="HP",
+          width="80",
+          height="80",
+          hi_def=True,
+          loading="lazy",
+          attrs={"class": "u-hide--small u-hide--medium"}
+          ) | safe
+        }}
+        <p>HP sells a wide range of Canonical-certified Ubuntu laptops and desktop machines globally.</p>
+        <p><a href="">See the Ubuntu certified AI-ready workstations&nbsp;&rsaquo;</a></p>
+      </div>
+      <a href="https://certification.ubuntu.com/desktop/models?vendors=HP" class="p-button--positive p-card__cta"><span>View HP with Ubuntu</span></a>
```

Comment:
Links to https://ubuntu.com/certified?category=Desktop&category=Laptop on the doc